java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -0,0 +1,66 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.RequestRoutingConfiguration.Route;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Server Management Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=server">Server Management Commands</a>
+ */
+public interface ServerManagementClusterCommands {
+
+    /**
+     * Get information and statistics about the Redis server. DEFAULT option is assumed. The command
+     * will be routed to all primaries.
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details. {@link
+     *     InfoOptions.Section#DEFAULT} option is assumed.
+     * @return Response from Redis cluster with a <code>String</code> containing the information for
+     *     the sections requested.
+     */
+    CompletableFuture<ClusterValue<String>> info();
+
+    /**
+     * Get information and statistics about the Redis server. DEFAULT option is assumed
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details.
+     * @param route Routing configuration for the command. Client will route the command to the nodes
+     *     defined.
+     * @return Response from Redis cluster with a <code>String</code> with the requested Sections.
+     *     When specifying a <code>route</code> other than a single node, it returns a dictionary
+     *     where each address is the key and its corresponding node response is the value.
+     */
+    CompletableFuture<ClusterValue<String>> info(Route route);
+
+    /**
+     * Get information and statistics about the Redis server. The command will be routed to all
+     * primaries.
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details.
+     * @param options - A list of {@link InfoOptions.Section} values specifying which sections of
+     *     information to retrieve. When no parameter is provided, the {@link
```

Comment:
Should be mentioned that it's being sent to all primaries and that the return type is a map